### ISSUE TITLE
Is MUI developing stopped?

### ISSUE THREAD

**Issue by win32nipuh** [firstfloorsoftware/mui#191]
Is this project is developing or it is dead?

**Comment by Krokoklemme** [comment]
well, the last pushes are 9, 11 and 24+h ago, so: no, not yet completely

**Comment by Krokoklemme** [comment]
but then, the last commit is from May 2016 🤔🤔

**Comment by Rod-in-NM** [comment]
I get the feeling that it isn't being supported anymore.

**Comment by DoCode** [comment]
Yes, I am also, and my startup, are very confused about this project.
Is the development now stopped?
What about new features or issues?
What about a V2?

**Comment by rynm** [comment]
I actively use this shell in a company product. It is quite useful as is.
There are small modifications that you would want to make for your
purposes. I myself have something that I still have to push to improve it.

**Comment by weitzhandler** [comment]
I hope it's not dead. I just PR'ed support for .NET Core 3 (#195).
I'm curious if @firstfloorsoftware is around at all.

**Comment by weitzhandler** [comment]
After rechecking the existing PRs, this project looks pretty dead. Shame.

**Comment by ameaninglessname** [comment]
> 
> 
> After rechecking the existing PRs, this project looks pretty dead. Shame.

This project is the first amazing WPF demo I'v ever seen, though, I have only half year of experience on WPF...

There is "probably a fix" that I did not PR yet. It is fixed maybe a year ago...

**Comment by weitzhandler** [comment]
I'm now working on a WPF project and of all the themes I've seen I like this one the most.
However, my project is .NET Core 3, so I guess if my PR (#196) isn't accepted or referred to by one of the maintainers (if any), I'll be giving up and move to a different project 😒

**Comment by Rod-in-NM** [comment]
I've like to try [MahApps.Metro](https://mahapps.com/). It is actively supported. But I can't convince my coworkers to leave this for MahApps. 😠

**Comment by ameaninglessname** [comment]
> 
> 
> I'm now working on a WPF project and of all the themes I've seen I like this one the most.
> However, my project is .NET Core 3, so I guess if my PR (#195) isn't accepted or referred to by one of the maintainers (if any), I'll be giving up and move to a different project 😒

Why not fork this and make it great again😋

**Comment by ameaninglessname** [comment]
> 
> 
> I've like to try [MahApps.Metro](https://mahapps.com/). It is actively supported. But I can't convince my coworkers to leave this for MahApps. 😠

That's cool👍 but still not cooler than this, my personnally saying.

And indeed, that one is "energetic", but this guy... looks..."bad"

**Comment by weitzhandler** [comment]
Alright guys, [`ModernUI.WPFCore`](https://www.nuget.org/packages/ModernUI.WPFcore) (`ModernUI.WPF.Core` was taken).
MahApps is too 'material'istic to me. Not so ideal for a Windows data-centric productivity app in my taste.

**Comment by JohanLarsson** [comment]
There is a fork [here](https://github.com/GuOrg/Gu.Wpf.ModernUI)  but it is not ready for release. Many breaking changes left and need to add UI-tests.

**Comment by Rod-in-NM** [comment]
> Alright guys, [`ModernUI.WPFCore`](https://www.nuget.org/packages/ModernUI.WPFcore) (`ModernUI.WPF.Core` was taken).
> MahApps is too 'material'istic to me. Not so ideal for a Windows data-centric productivity app in my taste.

What do you mean by MahApps is too materialistic?

**Comment by weitzhandler** [comment]
> What do you mean by MahApps is too materialistic?

Actually I decided to take MahApps for another ride, and it's excellent in every aspect. I totally take back what I said.
